### ISSUE TITLE
on atom opening, keeps giving `elixir` could not be spawned

### ISSUE THREAD

**Issue by rymir** [iraasta/autocomplete-erlang#3]
```
'elixir' could not be spawned. Is it installed and on your path? 
If so please open an issue on the package spawning the process.
```

Atom 0.196.0
Elixir 1.0.4
Erlang 17.1

![screen shot 2015-05-03 at 16 56 29](https://cloud.githubusercontent.com/assets/2323918/7445688/61f6e0ca-f1b5-11e4-95d4-0d94e60bd610.png)

**Comment by wende** [comment]
Yes. For the package to work You've got to have elixir in PATH

**Comment by rymir** [comment]
Sorry, you mean in my system path? If so, it definitely is:

```
(17.1)[tex@texmbp.home]$ echo $PATH
/Users/tex/src/17.1/bin:/Users/tex/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin
:/usr/bin:/bin:/usr/sbin:/sbin
```

and

```
(17.1)[tex@texmbp.home]$ which elixir
/usr/local/bin/elixir
```

**Comment by wende** [comment]
Are You sure it is in path for the user starting atom? All it does is
spawning elixir script.exs command so it has to be something about path.
Are You starting atom with sudo? If yes than make sure that su has elixir
in system path too

Krzysztof Wende
@iraasta

**Comment by rymir** [comment]
Thanks, interesting enough if I run atom from a terminal it works:

```
open /Applications/Atom.app
```

Checked the running user and is the same in both cases.

**Comment by rymir** [comment]
I suppose that's the issue here https://github.com/AtomLinter/Linter/issues/150

**Comment by Gyllsdorff** [comment]
This is also a problem on windows but the cause is another "bug".

On windows [`spawn` can only execute .exe files.](https://github.com/joyent/node/issues/2318) Elixir is a .bat file. 

I "fixed" it in my local copy by changing [Line 14 in lib/alchemide/wrapper.coffee](https://github.com/iraasta/autocomplete-erlang/blob/master/lib/alchemide/wrapper.coffee#L14) from 
`ac  = spawn("elixir", array.reverse())` 
to 
`ac = spawn(process.env.comspec, ["elixir"].concat(array.reverse()))`

Of course this will only work on windows.

**Comment by wende** [comment]
@Gyllsdorff  Good call!
That's probably why atom is developing the BufferedProcess class. I was never sure what for it is.
I will look onto that

**Comment by wende** [comment]
Fixed in d6858a22f6965c09dfb36b615243a885d38828f5
